### PR TITLE
Slight rewrite of the Share Creation and Share Acceptance sections

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ The next step is for the Sending Server to additionally discover:
 
 The Sending Server MAY first preform blacklist and whitelist checks on the FQDN.
 
-If a finite whitelist of receiver servers exists on the sender side, then this list may already contain all necessary information about its OCM API version and URL.
+If a finite whitelist of receiver servers exists on the sender side, then this list may already contain all necessary information.
```
Comment (mickenordin) [inline README.md]: should probably be allowlist and denylist, here and below

**Review by mickenordin — CHANGES_REQUESTED** [review]

**Review by MahdiBaghbani — APPROVED** [review]

**Review by mickenordin — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -161,11 +161,32 @@ If `https://<fqdn>/.well-known/ocm` does not exist, the Receiving Server MAY ins
...
+* using [httpsig](https://datatracker.ietf.org/doc/html/draft-cavage-http-signatures-12)
+
+The Receiving Server MAY discard the notification if any of the following hold true:
+* the HTTP Signature is missing
+* the HTTP Signature is not valid
+* no keypair is trusted or discoverable from the FQDN part of the `sender` field in the request body
+* the keypair used to generate the HTTP Signature doesn't match the one trusted or discoverable from the FQDN part of the `sender` field in the request body
+* the Sending Server is denylisted
+* the Sending Server is not allowlisted
+* the Sending Party is not trusted by the Receiving Party (i.e. the Sending Party's OCM Address does not appear in the Receiving Party's addressbook)
```
Comment (glpatcern) [inline README.md]: What about extending this a bit to reinject what was removed in https://github.com/cs3org/OCM-API/pull/111 ?

Comment (michielbdejong) [reply README.md]: good point, I'll add an explicit mention of the relevance of invites here.

Comment (michielbdejong) [reply README.md]: done in [8c73300](https://github.com/cs3org/OCM-API/pull/113/commits/8c733004d8cee6789735553b11c4549c3e74d98c)

**Pull request merged**